src/compile/data/transforms.ts

```diff
@@ -45,6 +48,48 @@ export class FilterNode extends DataFlowNode {
   }
 }
 
+export class GeoJSONNode extends DataFlowNode {
+  public clone() {
+    return new GeoJSONNode(duplicate(this.fields), this.geojson, this.signal);
+  }
+
+  public static make(model: ModelWithField): GeoJSONNode {
+    const geo = model.reduceFieldDef((rel, fieldDef, channel) => {
+      if ((contains([X, Y], channel) && contains([LATITUDE, LONGITUDE], fieldDef.type))
```

Comment:
Per: https://github.com/vega/vega-geo/issues/2#issuecomment-320533717
"The two params don't have to be mutually exclusive. The transform could handle both in one shot." So we might as well allow it. Otherwise we have to figure out which one they meant...